src/openforms/prefill/__init__.py

```diff
@@ -221,10 +223,16 @@ def _set_default_values(
         default_value = component.get("defaultValue")
         glom_path = Path(plugin, attribute)
         prefill_value = glom(prefilled_values, glom_path, default=None)
+
         if prefill_value is None:
             logger.debug("Prefill value for component %r is None, skipping.", component)
             continue
 
+        # 1693: we need to normalize values according to the format expected by the
```

Comment:
I think it should because the variables code path calls `_set_default_values` too!

However, I should probably write a test for this to prove it :-)